DiceKeys/Views/Screens/DiceKeyPresent.swift

```diff
@@ -26,49 +26,56 @@ struct DiceKeyPresentNavigationFooter: View {
     var BottomButtonFractionalWidth: CGFloat {
         0.9 / CGFloat(BottomButtonCount)
     }
+    
+    var navBarContentHeight: CGFloat { geometry.size.height / 12 }
+    var navBarImageHeight: CGFloat { navBarContentHeight * 0.75 }
 
     var body: some View {
         let view = VStack {
         ZStack {
             HStack(alignment: .top) {
-                
+                Spacer()
```

Comment:
Why do we need this spacer? Let the icon take all of left space

Reply:
The footer is outside the icons.  It ensures there's space on the left edge of the footer and the right edge of the footer.